README.md

```diff
@@ -58,11 +58,11 @@ Arrays can be merged in three ways - prepending data, appending data, and comple
   - (( prepend ))
   ```
 
-- To merge the two arrays together (each index of the new array will be merged into the original, additionals appended),
+- To replace the first array with the second,
   ensure that the first element in the new array is <br>
 
   ```yml
-  - (( inline ))
+  - (( replace ))
```

Comment:
looks like you dropped the docs for (( inline )), but inline is still supported as an explicit thing?